spec/beef/extensions/websocket_spec.rb

```diff
@@ -1,6 +1,7 @@
 require 'rest-client'
 require 'core/main/network_stack/websocket/websocket'
-require 'em-websocket-client'
+require 'websocket-client-simple'
+require 'byebug'
```

Comment:
@grantrburgess  can we remove the byebug from here if its not in use for debugging?